RELEASES.md

```diff
@@ -0,0 +1,47 @@
+# Releases
+
+This document describes how Strimzi handles versioning, releases new versions and communicates about them.
+
+Strimzi uses [semantic versioning](https://semver.org/) as a versioning schema for the releases of all the components.
+
+## Release Cycle
+
+Strimzi is using a release schedule of 1/2 months for the "core" component, the cluster operator.
+The [GitHub milestones](https://github.com/strimzi/strimzi-kafka-operator/milestones) are used to track the features and bug fixes included in each release.
+
+The other components, under the Strimzi umbrella, don't have a specific release cycle but when having a new releases is decided on a case-by-case basis depending on the new features added, bugs and vulnerabilities fixed.
+
+## Release Process
+
+Every Strimzi component has a corresponding documentation about how to make a new release.
+For example, the release process for the cluster operator is described [here](https://github.com/strimzi/strimzi-kafka-operator/blob/main/development-docs/RELEASE.md).
+
+## Keeping Track Of Changes
+
+We provide a changelog that includes all changes for each version taking the history from the first release.
+It is available as part of the repository of each component, for example, here is the changelog for the [Strimzi cluster operator](https://github.com/strimzi/strimzi-kafka-operator/blob/main/CHANGELOG.md).
+
+Every closed issue and PR should be added to a GitHub Milestone that provides an overview of everything that will be included in the corresponding release.
+
+Lastly, every new release will come with a GitHub Release which provides the exact changelog for that release.
+
+## Get Notified For New Releases
+
+Every new release, for each component, is announced in several ways:
+
+* Email to the Strimzi users CNCF mailing list.
+* Strimzi CNCF Slack channel.
+* Twitter [account](https://twitter.com/strimziio).
```

Comment:
We don't do this really. I usually tweet about it because I did the video for new minor releases. But I don't think we really tweet for every release.

Reply:
I was pretty sure about it, but it seems I was wrong. I will remove this bullet point then.